CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - On Windows, drag and drop is now optional and must be enabled with `WindowBuilderExtWindows::with_drag_and_drop(true)`.
 - On Wayland, fix deadlock when calling to `set_inner_size` from a callback.
 - On macOS, add `hide__other_applications` to `EventLoopWindowTarget` via existing `EventLoopWindowTargetExtMacOS` trait. `hide_other_applications` will hide other applications by calling `-[NSApplication hideOtherApplications: nil]`.
+- On macOS - fix inverted horizontal scroll
```

Comment:
Could you 'push back' the new entry instead of inserting in the middle of a section?

Also reformat it to match other entries here.

```suggestion
- On macOS, fix inverted horizontal scroll.
```